transport_nantes/mobilito/views.py

```diff
@@ -725,7 +725,16 @@ def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
                 f"latlng={lat},{lng}&key={MAPS_API_KEY}"
             )
             response = requests.get(url)
-            return JsonResponse(response.json())
```

Comment:
You must info log all requests we send and all responses received.  After that, info log success, warn or error on non-success.

The purpose of logging is understanding what happened in production.  You do not have the ability to teleport to the user, back in time, with development tools, to find out what happened.

Reply:
Done in the new logger, added details into the reverse geocoding-specific log file.